scanner/indexer/indexer.go

```diff
@@ -19,6 +20,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote/transport"
 	"github.com/quay/claircore"
 	"github.com/quay/claircore/datastore/postgres"
+	ccIndexer "github.com/quay/claircore/indexer"
```

Comment:
`ccindexer`

I think it's usually preferred to keep it all lowercase like package names (I know other parts of the stackrox/stackrox codebase uses camelCase, but I don't think it's right)